manual/gce-sru-20.2.45.txt

```diff
@@ -0,0 +1,1857 @@
++ SRU_VARS=./sru-scripts/sru-vars.template
```

Comment:
I think this manual file should also contain the shellscript that generated the output provided here. For example:

https://github.com/cloud-init/ubuntu-sru/blob/master/manual/openstack-sru-20.2.45.txt#L1